src/plugins/so_management/server/routes/find.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { IRouter } from 'src/core/server';
+import { injectMetaAttributes } from '../lib';
+import { ISavedObjectsManagement } from '../services';
+
+export const registerFindRoute = (
+  router: IRouter,
+  managementServicePromise: Promise<ISavedObjectsManagement>
+) => {
+  router.get(
+    {
+      path: '/api/kibana/management/saved_objects/_find',
+      validate: {
+        query: schema.object({
+          perPage: schema.number({ min: 0, defaultValue: 20 }),
```

Comment:
I think this should be coming from `savedObjects:perPage`

Reply:
The `savedObjects:perPage` value is actually accessed and sent by the client.

https://github.com/elastic/kibana/blob/56571bac847b1579dbaca2d292614744a248d297/src/legacy/core_plugins/kibana/public/management/sections/objects/_objects.js#L60

The route's joi schema was not, and I did a rather straight forward migration to config-schema. As this is an internal API and to avoid adding another dependency to the route definition, I'm leaning to keep it that way, unless you want me to perform the change?

Reply:
We can do it later when moving `savedObjects:perPage` registration to saved_objects_management plugin.